src/libraries/Common/src/System/Security/Cryptography/DSASecurityTransforms.cs

```diff
@@ -19,397 +19,196 @@ public static new DSA Create()
         {
             return new DSAImplementation.DSASecurityTransforms();
         }
+    }
```

Comment:
Why is this being changed away from a nested type?

Reply:
I did it to match RSASecurityTransforms and to avoid the need for copied `#if`s in the partial files.

Reply:
OK, if RSA already did this then consistency makes sense.  My hesitation was "but then it's less consistent", which was incorrect.